conanfile.py

```diff
@@ -140,8 +144,12 @@ def layout(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.variables["MP_UNITS_BUILD_MODULES"] = self._build_modules
         tc.variables["MP_UNITS_BUILD_LA"] = self._build_all and not self._skip_la
         tc.variables["MP_UNITS_USE_LIBFMT"] = self._use_libfmt
+        if self._build_modules:
+            tc.variables["CMAKE_CXX_COMPILER_CLANG_SCAN_DEPS"] = "/usr/bin/clang-scan-deps-16"
+            tc.variables["CMAKE_EXPERIMENTAL_CXX_MODULE_CMAKE_API"] = "aa1f7df0-828a-4fcd-9afc-2dc80491aca7"
```

Comment:
This also probably should land in a toolchain file.

Reply:
<https://github.com/mpusz/mp-units/pull/350#discussion_r1336893398> is outdated but not resolved.